Octokit/Models/Request/NewAuthorization.cs

```diff
@@ -11,12 +11,35 @@ namespace Octokit
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public class NewAuthorization
     {
+        // TODO: I'd love to not need this
+        public NewAuthorization()
+        {
+        }
+
+        public NewAuthorization(string note, string[] scopes)
+        {
+            Scopes = scopes;
+            Note = note;
+        }
+
+        public NewAuthorization(string note, string[] scopes, string fingerprint)
+        {
+            Scopes = scopes;
```

Comment:
I guess we have the same problem with the property setter. We can maybe worry about this separately as a general problem and not in this PR.